src/aura/HH_AutoComplete/HH_AutoCompleteController.js

```diff
@@ -0,0 +1,23 @@
+({
+    handleNewContact: function(component, event, helper) {
+        var con = {'sobjectType': 'Contact'};
+        var autocomplete = component.find('autocomplete');
```

Comment:
You're using what the user searched for as the default for the name of their new contact?  Clever.  This is one of those subtle usability things that you go out of your way to do for users but that they probably never notice except subconsciously.